tests/js/spec/components/platformIconTile.spec.js

```diff
@@ -0,0 +1,21 @@
+import {
+  selectPlatfrom,
+  getColorStyles,
+  PlatformIconTile,
+} from 'app/components/platformIconTile';
+
+describe('platformIconList', function() {
+  it('should return appropriate color from selectPlatfrom for a platform when provided with object of colors', function() {
+    const platforms = ['javscript'];
+    const PLATFORM_COLORS = {javscript: ['#3060b8']};
+    expect(selectPlatfrom(PLATFORM_COLORS, platforms)).toEqual(['#3060b8']);
+  });
+  it('should return appropriate style from getColorStyles for a platform when provided with object of colors', function() {
+    const platforms = ['javscript'];
+    const arg = {monoTone: '', platforms};
+    expect(getColorStyles(arg).styles).toEqual('background-color:#625471;color:#fff;');
+  });
+  it('should return appropriate JSX', function() {
+    expect(PlatformIconTile).toMatchSnapshot();
```

Comment:
@billyvg are we still using snapshots?

Reply:
Yeah I'll need to write a eslint plugin for this, but let's use `toSnapshot()`